pages/_app.tsx

```diff
@@ -19,6 +19,8 @@ const App: FC<{
     <Head>
       <meta name="viewport" content="width=device-width, initial-scale=1" />
       <meta name="description" content="Show your collection to the world." />
+      <meta property="og:title" content="Gallery" key="og:title" />
```

Comment:
I moved `og:title` and `twitter:title` out of _document.tsx and into _app.tsx, because it looks like tags in `_document` don't get overridden even when you use the `key` identifier. So without doing that, the page ended up with two `og:title` tags - one set by _document and one set by NftDetailPage for example.

Next js docs recommended putting tags that can be page-specific in pages and components instead, which worked.


https://stackoverflow.com/questions/68204973/avoid-duplicate-meta-description-and-keywords-in-next-js
https://nextjs.org/docs/advanced-features/custom-document

Reply:
and the `key` identifier is used to prevent duplicate tags.